packages/calypso-css-custom-properties/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "@automattic/calypso-css-custom-properties",
+	"version": "1.0.0-alpha.0",
+	"description": "Calypso Shared Style Bits",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"calypso",
+		"wordpress",
+		"sass",
+		"css"
+	],
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/master/packages/calypso-css-custom-properties/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/wp-calypso.git"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"engines": {
+		"node": "^10.13.0",
+		"npm": "^6.1.0"
+	},
+	"main": "dist/custom-properties.css",
+	"dependencies": {
+		"color-studio": "github:automattic/color-studio#1.0.1"
+	},
+	"devDependencies": {
+		"node-sass": "^4.11.0"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
+		"build": "node bin/build-custom-properties-css.js",
+		"postinstall": "node bin/postinstall.js"
```

Comment:
we probably want a `clean` too to remove the built custom props, and to call `lerna run clean` in the root `clean` script

Reply:
95577b33eddab41398976804bd7702d6846e51d9

(also used tabs in package.json, watch out for whitespace 🙈)

Reply:
Do you use editorconfig? :-)

https://github.com/Automattic/wp-calypso/blob/9073dcc3c49d2afadf602e042dd454779039ede1/.editorconfig#L20-L22

Reply:
Apparently I went the wrong way 😅 

Happy to do whatever folks prefer, shall I revert the space change here or make them all spaces?

Reply:
stick with spaces i think. One of these days we'll figure out why the tooling defaults to tabs on occasion.

Reply:
As of e2013a8 we might have this ready 😵